Gemfile

```diff
@@ -27,7 +27,7 @@ gem "mail-notify"
 gem "mlanett-redis-lock"
 gem "momentjs-rails"
 gem "mongo"
-gem "mongoid"
+gem "mongoid", "8.1.4"
```

Comment:
Just wondering, are we deliberately restricting this to 8.1.4 or was this imposed by a different package? If it is our requirement, then would a comment help to explain that?